content/kubermatic/master/tutorials-howtos/external-clusters/aks/_index.en.md

```diff
@@ -39,29 +60,64 @@ You can click on `Machine Deployments` to get the details:
 
 ## Update Cluster
 
-### Upgrade Version
+### Upgrade Cluster Version
 
 When an upgrade for the cluster is available, a little dropdown arrow will be shown beside the `Control Plane Version` on the cluster’s page.
 To start the upgrade, choose the desired version from the list of available upgrade versions and click on `Change Version`.
 
+![Upgrade Available](/img/kubermatic/master/tutorials/external_clusters/aks_upgrade_available.png "Upgrade Available")
+
 ![Upgrade AKS](/img/kubermatic/master/tutorials/external_clusters/upgrade_aks.png "Upgrade AKS")
 
 If the version upgrade is valid, the cluster state will change to `Reconciling`.
 
-### Scale the Machine Deployment
+## Edit the Machine Deployment
+
+{{% notice info %}}
+Only one operation can be performed at one point of time. If replica is updated then upgrade kuberntes version will be disabled and vice versa.
+{{% /notice %}}
+
+- Navigate to the cluster overview, scroll down to machine deployments.
 
-Navigate to the cluster overview, scroll down to machine deployments and click on the edit icon next to the machine deployment you want to edit.
+- Click on the edit icon next to the machine deployment you want to edit.
 
 ![Update AKS Machine Deployment](/img/kubermatic/master/tutorials/external_clusters/edit_md.png "Update AKS Machine Deployment")
 
-In the popup dialog, you can now increase or decrease the number of worker nodes that are managed by this machine deployment.
-Either specify the number of desired nodes or use the `+` or `-` to increase or decrease node count.
+- Upgrade Kubernetes Version. Select the Kubernetes Version from the dropdown to upgrade the md.
```

Comment:
use full words instead of jus t`md`, customers may not have an idea what the `md` is.